wordpress-mobile/WordPress-iOS
# Removed client-side .blog subdomain handling

Fixes the user-facing portion of #10919.

To test:
- Checkout the branch and confirm that existing tests pass.

Update release notes:
